### PR TITLE
[CS1998] `ClusterHeartbeatReceiverSpec` This async method lacks `await` operators

### PR DESCRIPTION
**Pull request by eaba**
## Checklist

For significant changes, please ensure that the following have been completed (delete if not relevant):

* [x] This change follows the [Akka.NET API Compatibility Guidelines](https://getakka.net/community/contributing/api-changes-compatibility.html).
* [x] This change follows the [Akka.NET Wire Compatibility Guidelines](https://getakka.net/community/contributing/wire-compatibility.html).
* [x] I have [reviewed my own pull request](https://getakka.net/community/contributing/index.html#review-your-own-pull-requests).

### CODE REVIEW

File: src/core/Akka.Cluster.Tests/ClusterHeartbeatReceiverSpec.cs [hunk 1]
```diff
@@ -58,8 +59,8 @@ public async Task ClusterHeartbeatReceiver_should_write_correct_debug_messages_o
         {
             var heartbeater = Sys.ActorOf(ClusterHeartbeatReceiver.Props(Cluster.Get(Sys)));
 
-            EventFilter.Debug(contains: "- Sequence number [2]")
-                .ExpectOne(() => heartbeater.Tell(new Heartbeat(Cluster.Get(Sys).SelfAddress, 2, 3)));
+            await EventFilter.Debug(contains: "- Sequence number [2]")
+                .ExpectOneAsync(() => { heartbeater.Tell(new Heartbeat(Cluster.Get(Sys).SelfAddress, 2, 3)); return Task.CompletedTask; });
```
Comment (eaba) [inline src/core/Akka.Cluster.Tests/ClusterHeartbeatReceiverSpec.cs:63]: Change to `ExpectOneAsync(Func<Task> action, CancellationToken cancellationToken = default)`
https://github.com/akkadotnet/akka.net/blob/0bf90827082b46b8ef7363e291d84934194c80fb/src/core/Akka.TestKit/EventFilter/Internal/EventFilterApplier.cs#L78-L87

File: src/core/Akka.Cluster.Tests/ClusterHeartbeatReceiverSpec.cs [hunk 2]
```diff
@@ -68,8 +69,8 @@ public async Task ClusterHeartbeatSender_should_write_correct_debug_messages_on_
             var heartbeater = Sys.ActorOf(Props.Create(() => new ClusterHeartbeatSender(Cluster.Get(Sys))));
             heartbeater.Tell(new ClusterEvent.CurrentClusterState());
             
-            EventFilter.Debug(contains: "- Sequence number [2] - Creation time [00:00:03]")
-                .ExpectOne(() => heartbeater.Tell(new HeartbeatRsp(Cluster.Get(Sys).SelfUniqueAddress, 2, 3000000000)));
+            await EventFilter.Debug(contains: "- Sequence number [2] - Creation time [00:00:03]")
+                .ExpectOneAsync(() => { heartbeater.Tell(new HeartbeatRsp(Cluster.Get(Sys).SelfUniqueAddress, 2, 3000000000)); return Task.CompletedTask; });
```
Comment (eaba) [inline src/core/Akka.Cluster.Tests/ClusterHeartbeatReceiverSpec.cs:73]: Change to `ExpectOneAsync(Func<Task> action, CancellationToken cancellationToken = default)`
https://github.com/akkadotnet/akka.net/blob/0bf90827082b46b8ef7363e291d84934194c80fb/src/core/Akka.TestKit/EventFilter/Internal/EventFilterApplier.cs#L78-L87

**Review by eaba — COMMENTED** [review]
Changes

**Review by Aaronontheweb — APPROVED** [review]
LGTM

**Pull request merged**